src/app/components/AddFavorites/AddFavorites.js

```diff
@@ -0,0 +1,20 @@
+import styles from "./AddFavorities.moudle.scss";
+import star from "../../../Img/star.svg";
+import starFilled from "../../../Img/starFilled.svg";
+
+export const AddFavorite = ({ favorite, setFavorite, id }) => {
+  return (
+    <div className={styles.wrapper}>
+      <img
+        alt="star"
```

Comment:
This `alt` should rather be "add to favorites" or "remove from favorites" depending on current state.